docs/core/integration-quality-scale/rules/iqs007.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "IQS007: Check during integration initialization if we are able to set it up correctly"
+---
+
+IQS007 is as follows:
+
+Check during integration initialization if we are able to set it up correctly
+
+## Reasoning
+
+When we initialize an integration, we should check if we are able to set it up correctly.
+Since not every integration polls for data, for example when they only expose services, we should check so we can let the user know that it doesn't work.
```

Comment:
I don't think we should mention an example. Also for polling integrations we want to poll once during setup to check that it works.
```suggestion
This way we can immediately let the user know that it doesn't work.
```